src/v2/index.js

```diff
@@ -92,6 +92,7 @@ graphQLServer.use('*', helmet({
   frameguard: false,
   noSniff: false,
   xssFilter: false,
+  contentSecurityPolicy: false,
```

Comment:
Why did you have to add this? If it was just for development remove from this PR.
```suggestion
```

Reply:
was getting some csp errors initially.